pkg/minio/minio.go

```diff
@@ -44,6 +44,7 @@ const (
 type Minio struct {
 	Clientset kubernetes.Interface
 	Namespace string
+	cli       *miniogo.Client
```

Comment:
I'd call this `client`. I think `cli` is confusing since that is most often referred to for command line interface